receptorctl/noxfile.py

```diff
@@ -28,12 +29,47 @@ def install(session: nox.Session, *args, req: str, **kwargs):
     session.install("-r", requirements_directory / f"{req}.in", *args, **kwargs)
 
 
+@nox.session
+def version(session: nox.Session):
+    """
+    Create a .VERSION file.
+    """
+    with open(os.devnull, "w") as devnull:
+        try:
+            official_version = session.run_install(
+                "git",
+                "describe",
+                "--exact-match",
+                "--tags",
+                external=True,
+                stderr=devnull,
```

Comment:
Have you tried `subprocess.DEVNULL` which is what's customary to use with `subprocess.run()`? Accessing `os.devnull` seems like a low-level *NIX-only hack that I'd avoid.

Reply:
argh, I think I'm making you repeat yourself as this was mentioned in the previous comment. thanks for catching it, again... maybe I just got fixated, which I tend to do.

Reply:
No worries! It needs to be tested, though. Since nox's API is a wrapper around subprocess so I don't know how it'll behave..

Reply:
Oh, and have you considered using `silent=True` instead? https://nox.thea.codes/en/stable/config.html#nox.sessions.Session.run suggests that it suppresses any output unless the command fails which seems useful.

Reply:
thanks! yes, `subprocess.DEVNULL` seems to work as expected and is a lot simpler than that hack I cobbled together.

I had `silent=True` included but the output is a `fatal` message and the command fails so I guess it doesn't really fit a need.